addons/mail/static/src/messaging/component/message_list/message_list.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.messaging.component.MessageList" owl="1">
-        <div class="o_MessageList" t-att-class="{ 'o-empty': threadViewer and threadViewer.messages.length === 0, 'o-has-message-selection': props.selectedMessage }" t-on-scroll="_onScroll">
+        <div class="o_MessageList" t-att-class="{ 'o-empty': threadViewer and threadViewer.messages.length === 0, 'o-has-message-selection': props.selectedMessageLocalId }" t-on-scroll="_onScroll">
```

Comment:
prop in template (same below), maybe make selectedMessage entity getter

Reply:
I wouldn't just because this would mean any use of this getter should be guarded, especially when this props is forwarded from discuss > thread viewer > message list > message.